kinetic/distribution.yaml

```diff
@@ -7718,6 +7718,22 @@ repositories:
       url: https://github.com/larics/smartek_camera.git
       version: master
     status: maintained
+  soem:
+    doc:
+      type: git
+      url: https://github.com/smits/soem.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/smits/soem-gbp.git
+      version: 1.3.0-0
+    source:
+      test_pull_requests: true
```

Comment:
I've followed the instructions on http://wiki.ros.org/buildfarm/Pull%20request%20testing, I've manually added the webhook so it should be good to go.